c++/src/kj/async-inl.h

```diff
@@ -1985,36 +1985,14 @@ class XThreadEventImpl<Func, Promise<T>> final: public XThreadEvent {
   friend Executor;
 };
 
-}  // namespace _ (private)
-
-template <typename Func>
```

Comment:
I moved these down below so that all the Executor member functions are all defined together. I'm happy to leave them up here, too, though.

Reply:
I think I'll move them back, since Executor's function implementations are already a bit spread out, I now see.